functions/use_version.go

```diff
@@ -0,0 +1,82 @@
+package functions
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"os"
+	"path/filepath"
+	"runtime"
+)
+
+var binPath, filePath string
+
+func createDirIfNotExists() {
+	homeDir, err := os.UserHomeDir()
+	if err != nil {
+		log.Fatal(err)
+	}
+	binPath = filepath.Join(homeDir, ".netmaker", "bin")
+	if err := os.MkdirAll(binPath, os.ModePerm); err != nil {
+		log.Fatal(err)
+	}
+}
+
+func downloadVersion(version string) {
+	res, err := http.Get(fmt.Sprintf("https://github.com/gravitl/netclient/releases/download/%s/netclient-%s-%s", version, runtime.GOOS, runtime.GOARCH))
```

Comment:
if it is windows, the netclient binary will end in .exe (netclient-windows-amd64.exe)

Reply:
done